app/grandchallenge/components/models.py

```diff
@@ -514,6 +517,11 @@ def get_file_mimetypes(cls, *, kind):
                 "application/x-sqlite3",
                 "application/vnd.sqlite3",
             )
+        elif kind == InterfaceKind.InterfaceKindChoices.OBJ:
+            return (
+                "text/plain",
+                "application/octet-stream",
+            )
```

Comment:
What file did you test this with?

Reply:
OK, I'm waiting to get access to the 3D Teeth Seg dataset, we should double check with those.

Reply:
I tested with a bunch of files from the 3D teeth set and they are all of mimetype `text/plain`. The only file that was of mimetype `application/octet-stream` was a file that I had created myself yesterday, so I think we can safely ignore that then.